### PR TITLE
feat(leadspace): added ::part pseudo-element to component

### PR DESCRIPTION
**Pull request by guilhermelMoraes**
### Description

Added `::part` pseudo-element to `leadspace`

### Changelog

**New**

- `::part` pseudo element in `leadspace.ts`

### CODE REVIEW

File: packages/web-components/src/components/leadspace/leadspace.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
  * The LeadSpace component.
  *
  * @element dds-leadspace
+ * @csspart leadspace The first DOM node inside the shadow-root
```
Comment (asudoh) [inline packages/web-components/src/components/leadspace/leadspace.ts:27]:
```suggestion
 * @csspart section The first DOM node inside the shadow-root
```

File: packages/web-components/src/components/leadspace/leadspace.ts [hunk 2]
```diff
@@ -136,7 +137,7 @@ class DDSLeadSpace extends StableSelectorMixin(LitElement) {
   render() {
     const { gradientStyleScheme, type } = this;
     return html`
-      <section style="${this._getBackgroundImage()}" class="${this._getTypeClass()}">
+      <section style="${this._getBackgroundImage()}" class="${this._getTypeClass()}" part="leadspace">
```
Comment (asudoh) [inline packages/web-components/src/components/leadspace/leadspace.ts:140]:
```suggestion
      <section style="${this._getBackgroundImage()}" class="${this._getTypeClass()}" part="section">
```

**Review by asudoh — COMMENTED** [review]
Thanks @guilhermelMoraes - A suggestion given the context of "leadspace" seems to apply to the host element.

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @guilhermelMoraes for the update!

**Comment by jeffchew** [comment]
Hey @guilhermelMoraes , can you update snapshot files?

**Pull request merged**